server/routes/employer-sites.js

```diff
@@ -111,9 +112,15 @@ router.get(
     routeRedirect({ redirect: '/api/v1/employer-sites/details', match: 'employer-sites-detail' }),
   ],
   asyncMiddleware(async (req, res) => {
+    const { regions } = req.query;
     const { hcapUserInfo: user } = req;
 
-    const result = await getSitesForUser(user);
+    let result;
+    if (regions && user.isHA && user.regions.length > 0) {
```

Comment:
After looking at this again, I think I'd prefer having this logic on the FE, then we have a more understandable

```
if (HA)
  GET (/sites/region)
else 
  GET (/sites)
```
and we can more easily handle this authorization in middleware

Still just a suggestion and not a blocker

Reply:
Yeah I'd like it as just a separate endpoint and keep all the logic of which endpoint to hit on the FE.

Reply:
Fair enough